cmd/stfinddevice/main.go

```diff
@@ -15,17 +16,19 @@ import (
 	"github.com/syncthing/syncthing/lib/discover"
 )
 
+const defServer = "https://announce.syncthing.net/?insecure&noannounce"
```

Comment:
You're like my bad consciousness, forcing me to do things properly when I'd rather just gloss over them...

```
jb@syno:~/src/github.com/syncthing/syncthing git:(upstream ⚡ tlsdisco) 6M $ ./bin/stfinddevice I6KAH76-66SLLLB-5PFXSOA-UFJCDZC-YAOMLEK-CP2GB32-BV5RQST-3PSROAU
v4-1.discover.syncthing.net (308.327604ms):
  address: tcp://213.114.1.152:22001
  relay: relay://194.126.249.21:22067 (100 ms)
v6-1.discover.syncthing.net (337.064753ms):
  address: tcp://[2001:470:deeb:32::2bc]:22001
  relay: relay://194.126.249.21:22067 (100 ms)
v4-2.discover.syncthing.net (651.753514ms):
  address: tcp://213.114.1.152:22001
  relay: relay://194.126.249.21:22067 (100 ms)
v6-2.discover.syncthing.net (681.112743ms):
  address: tcp://[2001:470:deeb:32::2bc]:22001
  relay: relay://194.126.249.21:22067 (100 ms)
v6-3.discover.syncthing.net (1.132438097s):
  address: tcp://[2001:470:deeb:32::2bc]:22001
  relay: relay://194.126.249.21:22067 (100 ms)
v4-3.discover.syncthing.net (1.401200148s):
  address: tcp://213.114.1.152:22001
  relay: relay://194.126.249.21:22067 (100 ms)
jb@syno:~/src/github.com/syncthing/syncthing git:(upstream ⚡ tlsdisco) 6M $ ./bin/stfinddevice P56IOI7-MZJNU2Y-IQGDREY-DM2MGTI-MGL3BXN-PQ6W5BM-TBBZ4TJ-XZWICQ2
v4-1.discover.syncthing.net (183.939283ms):
  404 Not Found
v6-1.discover.syncthing.net (213.096379ms):
  404 Not Found
v4-2.discover.syncthing.net (529.66468ms):
  404 Not Found
v6-2.discover.syncthing.net (589.77936ms):
  404 Not Found
v6-3.discover.syncthing.net (1.038033217s):
  404 Not Found
v4-3.discover.syncthing.net (1.357241876s):
  404 Not Found
jb@syno:~/src/github.com/syncthing/syncthing git:(upstream ⚡ tlsdisco) 6M $ ./bin/stfinddevice -server https://v6-3.discover.syncthing.net P56IOI7-MZJNU2Y-IQGDREY-DM2MGTI-MGL3BXN-PQ6W5BM-TBBZ4TJ-XZWICQ2
v6-3.discover.syncthing.net (759.775227ms):
  Get https://v6-3.discover.syncthing.net?device=P56IOI7-MZJNU2Y-IQGDREY-DM2MGTI-MGL3BXN-PQ6W5BM-TBBZ4TJ-XZWICQ2: x509: certificate is valid for syncthing, not v6-3.discover.syncthing.net
jb@syno:~/src/github.com/syncthing/syncthing git:(upstream ⚡ tlsdisco) 6M $ ./bin/stfinddevice -server https://v6-3.discover.syncthing.net?insecure P56IOI7-MZJNU2Y-IQGDREY-DM2MGTI-MGL3BXN-PQ6W5BM-TBBZ4TJ-XZWICQ2
v6-3.discover.syncthing.net (1.243466777s):
  404 Not Found
```